README.md

```diff
@@ -170,6 +170,164 @@ Return all products, serialized as:
 ]
 ```
 
+###Pets
+In order to receive any pet information you must know the pets' owner id
+The base URL You will be user is ```/api/v1/users/:user_id/pets```
```

Comment:
No reason to correct it now, but on our next doc update let's clean up this copy/paste grammar bomb.